docs/why.md

```diff
@@ -1,5 +1,10 @@
 # Django's Atomic
 
+This doc will discuss the behaviours available through Django's `atomic`,
+and the outcomes people are usually trying to achieve with it.
+It goes on to outline some pitfalls that can result from using `atomic`,
+and how Subatomic avoids them.
```

Comment:
I think the commas in the middle of these sentences are unnecessary (and arguably incorrect because they create fragments that aren't self-contained clauses).